pkgs/test/lib/src/runner/browser/static/run_wasm_chrome.js

```diff
@@ -7,15 +7,41 @@
 // affect chrome.
 (async () => {
   // Fetch and compile Wasm binary.
-  let data = document.getElementById('WasmBootstrapInfo').dataset;
-  let modulePromise = WebAssembly.compileStreaming(fetch(data.wasmurl));
+  let data = document.getElementById("WasmBootstrapInfo").dataset;
 
   // Instantiate the Dart module, importing from the global scope.
-  let dart2wasm = await import('./' + data.jsruntimeurl);
-  let dartInstance = await dart2wasm.instantiate(modulePromise, {});
+  let dart2wasmJsRuntime = await import("./" + data.jsruntimeurl);
 
-  // Call `main`. If tasks are placed into the event loop (by scheduling tasks
-  // explicitly or awaiting Futures), these will automatically keep the script
-  // alive even after `main` returns.
-  await dart2wasm.invoke(dartInstance);
+  // Support three versions of dart2wasm:
+  //
+  // (1) Versions before 3.6.0-167.0.dev require the user to compile using the
+  // browser's `WebAssembly` API, the compiled module needs to be instantiated
+  // using the JS runtime.
+  //
+  // (2) Versions starting with 3.6.0-167.0.dev added helpers for compiling and
```

Comment:
It would be OK to drop support for a strategy that was only in a series of dev releases, but supporting it sounds good too.

After we have any other reason to bump the min SDK we will get this cleaned up to only the version 3 approach.